elan/rpc/rclient.go

```diff
@@ -72,11 +72,11 @@ func (r *remoteClient) UploadIfMissing(entries []*uploadinfo.Entry) error {
 	return err
 }
 
-func (r *remoteClient) BatchDownload(digests []digest.Digest, compressors []pb.Compressor_Value) (map[digest.Digest][]byte, error) {
+func (r *remoteClient) BatchDownload(digests []digest.Digest) (map[digest.Digest][]byte, error) {
 	defer observeTime(time.Now(), "BatchDownload")
 	ctx, cnx := context.WithTimeout(context.Background(), time.Minute*5)
 	defer cnx()
-	return r.c.BatchDownloadCompressedBlobs(ctx, digests, compressors)
+	return r.c.BatchDownloadBlobs(ctx, digests)
```

Comment:
Oh wait, ofc, we aren't even hitting that old endpoint anymore, we will change the `BatchDownloadBlobs` to mirror `BatchDownloadCompressedBlobs` but checking on the digests.

Reply:
Going off memory, might have slightly deviated. But looking at https://github.com/peterebden/remote-apis-sdks/blob/remove-withbalancername/go/pkg/client/cas.go#L656 , if we don't specify compressors (which is equivalent to specifying identity) it will populate the digest field instead of requests. We can't change it to read the compressor from the request because that shares the same field number as our addition.